libraries/standard/mqtt/src/mqtt.c

```diff
@@ -22,15 +22,25 @@
 #include <string.h>
 
 #include "mqtt.h"
+#include "private/mqtt_internal.h"
 
-/*-----------------------------------------------------------*/
-
-static int32_t sendPacket( MQTTContext_t * pContext, size_t bytesToSend )
+static int32_t sendPacket( MQTTContext_t * pContext,
```

Comment:
This signature is not very clear - Is it possible to make it the following:

```c
static int32_t sendPacket( MQTTContext_t * pContext, const uint8_t * pBufferToSend, size_t bytesToSend );
```

The callers can then pass either `pContext->networkBuffer.pBuffer` or `pPublishInfo->pPayload` for the second parameter. 

Also please write doc for this function.

Reply:
Sure. Thanks.